CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Recent and upcoming changes to lightdash
 ## Unreleased
 ### Fixed
  - bug where long lists of table were not scrollable in the side bar
+ - handle empty chart
```

Comment:
The changelog should be really user friendly, so more about what the benefit is to them:

> improved chart UI when there isn't enough data to create a chart